File: GeekBand/python/isUnique.py
# -*- coding: utf-8 -*-
"""
 * @file     isUnique.py
 * @brief    
 * @author   YongDu
 * @date     2021-07-25
"""


class Solution:
    def isUnique(self, astr: str) -> bool:
        return True if (len(set(astr)) == len(astr)) else False

File: GeekBand/javascript/isUnique.js
/**
 * @param {string} astr
 * @return {boolean}
 */
var isUnique = function (astr) {
    const map = {};
    for (const c of astr) {
        const v = map[c];
        if (v) { return false; }
        map[c] = 1;
    }
    return true;
};